fcomm/src/snapshots/fcomm__test__snapshot_serialized_expressions-2.snap

```diff
@@ -0,0 +1,9 @@
+---
+source: fcomm/src/lib.rs
+expression: input
+---
+{
+  "expr": {
+    "Source": "(+ 1 2)"
```

Comment:
Observation: this source data is 'code', so I would expect it to be passed either to  a function which explicitly evaluates, or to be used as input to a test of general evaluation (which is how the `fcomm` examples use it).

Reply:
And that would be why we actually run the fcomm examples in testing.